csharp.html.markdown

```diff
@@ -33,7 +36,7 @@ using System.Threading.Tasks;
 using System.IO;
 
 // defines scope to organize code into "packages"
-namespace Learning
+namespace LearningXInYMinutes.CSharp
```

Comment:
You included a '.' character here. Does that change anything at all? Does that need a little description to explain what it does?